index.html

```diff
@@ -1,30 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
the meta tag doesn't need the '/' which you have added at the end.

Reply:
xhtml uses a trailing'/', perhaps the editor is in that mode?
In any case, the classes uses html and not xhtml.